src/ontology/uberon-edit.obo

```diff
@@ -222480,10 +222480,20 @@ name: coracoid process of calcaneus
 def: "At the dorsal part of the base of the calcaneal tuberoisty is a pointed process called the coracoid process." [https://link.springer.com/article/10.1007/s00276-003-0220-0, https://www.imaios.com/en/vet-Anatomy/Vet-Anatomical-Part/Coracoid-process2]
 synonym: "processus coracoideus of calcaneus" EXACT []
 is_a: UBERON:0005913 ! zone of bone organ
-relationship: dc-contributor http://orcid.org/0000-0003-2699-3066
+relationship: dc-contributor https://orcid.org/0000-0003-2699-3066
 relationship: part_of UBERON:7500094 ! tubercle of calcaneus
 property_value: dcterms-date 2022-05-19T21:16:42Z xsd:dateTime
 
+[Term]
+id: UBERON:7500107
+name: sphenovomerine suture
+def: "Articulation of sphenoid bone with alae of vomer forms a schindylesis type of joint." [DOI:10.1097/SCS.0000000000002945, PMID:27483100]
```

Comment:
Consider rewording definition to align with sibling classes, e.g., something like
 "A suture joining the rostrum of sphenoid bone with the alae of vomer to form a schindylesis type of joint."

Reply:
What is "rostrum of sphenoid bone"? Is it sufficient to just say "sphenoid bone" here?

Reply:
I think it is sufficient to just have "sphenoid bone". I used terminology taken from the dbxref you provided.

From PMID:27483100:
"Abstract
Purpose: **Articulation of rostrum of sphenoid bone with alae of vomer forms a schindylesis type of joint. The circumference of this joint, called sphenovomerine suture (SVS)**, is very important in establishing a reliable surgical field in the endoscopic transsphenoidal pituitary surgery. Because of its vital role in endoscopic transsphenoidal pituitary surgery, this radio-anatomical study was designed to establish the morphological properties of SVS."